controllers/repository_test.go

```diff
@@ -29,6 +30,36 @@ func TestCreateRepository(t *testing.T) {
 		Do(validateCreateRepositoryInput).
 		Return(&ecr.CreateRepositoryOutput{}, nil)
 
+	validateGetRepositoryPolicyInput := func(input *ecr.GetRepositoryPolicyInput) {
```

Comment:
[nit] ditto - I'm not sure we why aren't using this pattern:
```go
gomock.EXPECT().
    GetRepositoryPolicy(input).
    Return(...)
```

Reply:
Not sure? I also thought it was odd.